### DISCUSSION TITLE
What is the default settings file content?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#727] [Q&A] [answered]
I can't find the default settings file content, I messed up mine so I ended up deleting everything in it. I'm using the built in WinGet inside Windows insider.
I downloaded the source code but couldn't find it in there.

**Comment by denelon** [comment, marked as answer]
If you have vscode as your default editor, there is context aware help with the JSON schema.

Here is my file:

```JSON
// For documentation on these settings, see: https://aka.ms/winget-settings
{ 
    "$schema": "https://aka.ms/winget-settings.schema.json",

    "source": {"autoUpdateIntervalInMinutes": 5},

    "visual": {"progressBar": "rainbow"},

    "experimentalFeatures": {
        "experimentalCmd": true,
        "experimentalArg": true,
        "experimentalMSStore": true,
        "list": true,
        "upgrade": true,
        "uninstall": true
    }
}
```

**Reply by ghost** [reply]
Thank you, that's perfect, I don't even need to change anything, has got all the right options.
